RecoTracker/MeasurementDet/interface/MeasurementTrackerEvent.h

```diff
@@ -41,11 +41,8 @@ class MeasurementTrackerEvent {
 
    MeasurementTrackerEvent(const MeasurementTrackerEvent & other) = delete;
    MeasurementTrackerEvent & operator=(const MeasurementTrackerEvent & other) = delete;
-   MeasurementTrackerEvent(MeasurementTrackerEvent && other) = delete;
-   MeasurementTrackerEvent & operator=(MeasurementTrackerEvent && other) = delete;
-
-   //  for the edm wrapper...
-   void swap(MeasurementTrackerEvent &other) ;
+   MeasurementTrackerEvent(MeasurementTrackerEvent && other);
```

Comment:
This does not work. `MeasurementTrackerEvent` may own the pointers it holds. In that case, the default move constructor will NOT set the pointers to nullptr in the moved object. This will result in the pointed to objects being multiply deleted.

Reply:
It actually works because the `~MeasurementTrackerEvent()` deletes the pointers only if `theOwner == true`
https://github.com/cms-sw/cmssw/blob/46354df3148f7d63c56909a3326f57af0343ce25/RecoTracker/MeasurementDet/src/MeasurementTrackerEvent.cc#L4-L9
which is correctly set to `false` for `other`.

But if you want, I can set the pointers of `other` to `nullptr` as well as a precaution.

Reply:
No it is fine. I withdraw my objection.